tests/resolvers/Mutation/createEvent.spec.ts

```diff
@@ -166,18 +185,18 @@ describe("resolvers -> Mutation -> createEvent", () => {
         organizationId: testOrganization?.id,
         allDay: false,
         description: "newDescription",
-        endDate: new Date().toUTCString(),
+        endDate: new Date("2023-01-29T00:00:00Z"),
         endTime: new Date().toUTCString(),
         isPublic: false,
         isRegisterable: false,
         latitude: 1,
         longitude: 1,
         location: "newLocation",
         recurring: false,
-        startDate: new Date().toUTCString(),
+        startDate: new Date("2023-01-01T00:00:00Z"),
```

Comment:
Will this test work after `2023-01-29T00:00:00Z`?

Reply:
Same goes to this